test/scripts/install-contour-working.sh

```diff
@@ -71,11 +69,13 @@ for file in ${REPO}/examples/contour/02-job-certgen.yaml ${REPO}/examples/contou
   # we built above.
   sed \
     "-es|imagePullPolicy: Always|imagePullPolicy: IfNotPresent|" \
-    "-es|image: ghcr.io/projectcontour/contour:.*$|image: ghcr.io/projectcontour/contour:${VERSION}|" \
+    "-es|image: ghcr.io/projectcontour/contour:.*$|image: ghcr.io/projectcontour/contour:latest|" \
     "$file" | \
   ${KUBECTL} apply -f -
 done
 
+${KUBECTL} patch deployment -n projectcontour contour --patch '{"spec": {"template": {"metadata": {"annotations": {"timestamp": "'$(date +%s)'"}}}}}'
```

Comment:
There could be small comment to explain the purpose: patching the deployment with timestamp will trigger pod restart.